.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+* **Please check if the pull request fulfills these requirements**
+  
+- [ ] I have read the CONTRIBUTING guidelines 
+- [ ] The commit message follows our guidelines
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+
+* **What is the current behavior?** (You can also link to an open issue here)
```

Comment:
Just noticed the American spelling of behaviour. It is also like this in `epiparameter`, no need to change.

Reply:
Changed.